### PR TITLE
ticket #1396: disallow duplicated page paths

### PR DESCRIPTION
**Pull request by alexanderadam**
Hi @iphilgood,

this PR disallows duplicated page paths.

Thank you very much,
Alex

### CODE REVIEW

File: spec/models/page_spec.rb [hunk 1]
```diff
@@ -421,14 +429,14 @@
         expect(child.path).to eq('/parent/child')
       end
     end
-  end
+  end # path
```
Comment (iphilgood) [inline spec/models/page_spec.rb]: @alex is this comment necessary? :wink:

File: spec/models/page_spec.rb [hunk 2]
```diff
@@ -456,7 +464,7 @@
     it 'should be findable via scope' do
       expect(Qbrick::Page.by_identifier(cat_page.identifier)).to eq(cat_page)
     end
-  end
+  end # translated
```
Comment (iphilgood) [inline spec/models/page_spec.rb]: @alexanderadam same here :wink:

**Comment by iphilgood** [comment]
@alexanderadam just have a look at my comments, otherwise good work :raised_hands:

File: spec/models/page_spec.rb (hunk 1 above)
Comment (alexanderadam) [inline spec/models/page_spec.rb]: not necessary but it helps to navigate in files with long blocks to know where the `end` belongs to.

**Pull request merged**